code-for-canada/project-thundercat
# [Look and Feel] Progress Bar

# Description

Creation of generic ProgressPane and ProgressNode to show progress through a sequence of instruction pages. Also stubbed SideNavigation component for the next task

## Type of change

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

Please provide if applicable (browser for `frontend` or any visual way to show a `backend` change).
![image](https://user-images.githubusercontent.com/2746350/53196121-308f6e80-35e5-11e9-8cab-de70b729983c.png)
![image](https://user-images.githubusercontent.com/2746350/53196131-3be29a00-35e5-11e9-81f4-0c5ed258cfb1.png)

![image](https://user-images.githubusercontent.com/2746350/53196152-4a30b600-35e5-11e9-9af9-7d72bb3db7c1.png)
![image](https://user-images.githubusercontent.com/2746350/53196159-5288f100-35e5-11e9-8b48-801c9a8ad9b0.png)

![image](https://user-images.githubusercontent.com/2746350/53196171-5ae12c00-35e5-11e9-82ea-a56f5067dc59.png)

![image](https://user-images.githubusercontent.com/2746350/53196181-63d1fd80-35e5-11e9-8fe6-7844a7ac11eb.png)
![image](https://user-images.githubusercontent.com/2746350/53196192-6df3fc00-35e5-11e9-857b-487cf2d56046.png)
![image](https://user-images.githubusercontent.com/2746350/53196204-764c3700-35e5-11e9-80d6-424f661d55f4.png)


# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to EmibSample Test
2. Page through instruction panes (currently subbed)
3. Ensure progressbar updates; buttons show correct text; and layout matches the look and feel

Screenshot of unit tests run passing:
![image](https://user-images.githubusercontent.com/2746350/53188215-e43b3300-35d2-11e9-8fee-064dc33a5c60.png)

# Checklist

Applicable for all code changes.
